### PR TITLE
Enable --before for global scripts

### PR DESCRIPTION
**Pull request by mkam039**
When you pass model selection of `--before` flag, global scripts are not run.

### CODE REVIEW

File: src/fal/cli.py [hunk 1]
```diff
@@ -235,11 +235,12 @@ def _run(
         run_scripts(scripts, project)
 
         # then run global scripts
-        global_scripts = list(
-            map(
-                lambda path: FalScript(None, path, []),
-                faldbt._global_script_paths,
+        if selector_flags is None and before is False:
```
[the remark this answers is not in the document]
Comment (mkam039) [reply src/fal/cli.py:238]: Do you mean select a global script? We don't support running just global scripts. That would ideally be a different flag, right? `fal run --global`

[the remark this answers is not in the document]
Comment (chamini2) [reply src/fal/cli.py:238]: But let's say, you want to run ONLY a global script? I don't know if it makes sense. I think it does.

[the remark this answers is not in the document]
Comment (mkam039) [reply src/fal/cli.py:238]: I think it should be done with something like `fal run --script my_global_script.py --global`

[the remark this answers is not in the document]
Comment (chamini2) [reply src/fal/cli.py:238]: Interesting. I think in the future we will have to revisit the whole before/after/global concepts, but I'm OK with this for now.

[the remark this answers is not in the document]
Comment (chamini2) [reply src/fal/cli.py:238]: I would like a 3rd pair of eyes here for opinions on the subject, though. Can we leave it open until others check it?

**Comment by mkam039** [comment]
@chamini2 this PR is updated to just include support for `--before` tag in global scripts

File: src/faldbt/parse.py [hunk 2]
```diff
@@ -75,7 +75,11 @@ def get_global_script_configs(source_dirs: List[Path]) -> List[str]:
                 if fal_config is not None:
                     # sometimes `scripts` can *be* there and still be None
                     script_paths = fal_config.get("scripts") or []
-                    global_scripts += script_paths
+                    if isinstance(script_paths, list):
+                        global_scripts['after'] += script_paths
+                    else:
+                        global_scripts['before'] += script_paths.get('before', [])
+                        global_scripts['after'] += script_paths.get('after', [])
```
Comment (chamini2) [inline src/faldbt/parse.py:82]: Use `script_paths.get('after') or []` instead
> sometimes `scripts` can *be* there and still be None

**Pull request merged**